docs/advanced_usage/10_continue.rst

```diff
@@ -1,14 +1,21 @@
 Continue
 ========
 
-SMAC automatically restores states where it left off if a run was interrupted or finished. To do so, it reads in old
-files (derived from scenario's name, output_directory and seed) and sets the components.
+SMAC can automatically restore states where it left off if a run was interrupted or prematurely finished. To do so, 
+it reads in old files (derived from scenario's name, output_directory and seed) and obtains the scenario information
+of the previous run from those to continue the run.
+
+The behavior can be controlled by setting the parameter ``overwrite`` in the facade to True or False, respectively:
+If set to True, SMAC overwrites the run results if a previous run is found that is consistent in the meta data with 
+the current setup. If set to False and a previous run is found that is consistent in the meta data, 
```

Comment:
Duplicated?

Reply:
Sorry, what is duplicated? If referring to the "If set to False" being there twice: this is on purpose, as there are two different behaviors in this case. I made this more clear with the enumeration structure.